app/views/DeploymentHistoryPage.scala.html

```diff
@@ -51,7 +51,7 @@ <h1>Deployments</h1>
                 <input type="hidden" id="selected-environment" value="@selectedEnv.asString">
                 <div class="col-xs-3">
                     <label for="search" title="Search across all fields" data-toggle="tooltip">Search</label>
-                    <input id="search" class="search form-control" type="text" name="search" value='@form("search").value' autofocus>
+                    <input id="search" class="search form-control" type="text" name="search" value='@form("search").value' autofocus autocomplete="off">
```

Comment:
why autocomplete=off?

Reply:
It stops the drop down of the search box covering the number of deployments